graphql/task_resolver.go

```diff
@@ -430,6 +431,15 @@ func (r *taskResolver) GeneratedByName(ctx context.Context, obj *restModel.APITa
 	return &name, nil
 }
 
+// ImageID is the resolver for the imageId field.
+func (r *taskResolver) ImageID(ctx context.Context, obj *restModel.APITask) (string, error) {
+	imageId, err := distro.GetImageIDFromDistro(ctx, *obj.DistroId)
+	if err != nil {
+		return "", InternalServerError.Send(ctx, fmt.Sprintf("unable to find imageId from distro: '%s'", err.Error()))
```

Comment:
lets have the error be more similar to our other errors like 
"error finding image ID from distro"